### ISSUE TITLE
get the location updates after each 1 minute?

### ISSUE THREAD

**Issue by usamamashkoor** [rekab-app/background_locator#91]
Hi, I want to create an app like Uber, and I want to get the location updates of the user after each 1 minute even if the app is running in the background is there a way to do this using your library?

**Comment by diezep** [comment]
Yes, **it's possible**. In properties to initialize, have an option to set the interval in **seconds** to update the ubication, but in**files mention that only works in Android**. 
![image](https://user-images.githubusercontent.com/38699812/85347459-3b4e6200-b4be-11ea-96f4-c59ca9514f3c.png)

**Comment by usamamashkoor** [comment]
@diezep Thanks for answering my question, can you please let me know how can I do this for both Android and iPhone?

**Comment by mehdok** [comment]
You can use interval just for Android, iOS don't use interval for sending location updates, it just send the location when there is new data available;
